lib/read_only/generic/README.md

```diff
@@ -0,0 +1,90 @@
+# Read-only Verification Library for Generic Fortran Code
+
+This library implements the [PSyData API](
+https://psyclone.readthedocs.io/en/latest/psy_data.html#read-only-verification-library-for-gocean)
```

Comment:
This link is for gocean. Is that deliberate?

Reply:
Nope - good catch, thanks!